pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -354,20 +353,8 @@ func (m SQSMonitor) getNodeInfo(instanceID string) (*NodeInfo, error) {
 	}
 
 	if m.CheckIfManaged {
-		if nodeInfo.AsgName == "" {
-			// If ASG tags are not propagated we might need to use the API
-			// to retrieve the ASG name
-			nodeInfo.AsgName, err = m.retrieveAutoScalingGroupName(nodeInfo.InstanceID)
-			if err != nil {
-				return nil, fmt.Errorf("unable to retrieve AutoScaling group: %w", err)
-			}
-		}
-		if nodeInfo.Tags[m.ManagedAsgTag] == "" {
-			// if ASG tags are not propagated we might have to check the ASG directly
-			nodeInfo.IsManaged, err = m.isASGManaged(nodeInfo.AsgName, nodeInfo.InstanceID)
-			if err != nil {
-				return nil, err
-			}
+		if _, ok := nodeInfo.Tags[m.ManagedTag]; !ok { // austin: wait, the presence of this should be a good sign...how did this ever work?
```

Comment:
The behavior seems correct to me now: If we say we only want NTH to manage tagged nodes (i.e., `m.CheckIfManaged` is true), and we find a node that does _not_ have the tag, we set `nodeInfo.IsManaged` to false. Otherwise, we leave `nodeInfo.IsManaged = true`, which means either 1) we want to manage only tagged nodes, and this is a tagged node, or 2) we don't care; manage all the nodes.

Can we remove the comment, or is there still some doubt about the correctness of this logic?

Reply:
No concerns, I just need to remember to remove my through processes from comments before committing 😞